yourgov/components/FormMobileFoodVendorPermit/UploadFileTable.tsx

```diff
@@ -0,0 +1,196 @@
+	/** Remove an uploaded file. */
+	removeFile: (document: Document) => void;
+	/** Data for upload documents table. */
+	documents: Document[];
+};
+
+export const UploadFileTable = forwardRef<
+	HTMLTableElement,
+	UploadFileTableProps
+>(function UploadFileTable(
+	{ ariaLabelledby, documents, openDrawer, removeFile, tableId },
+	ref
+) {
+	const error = false;
+	const loading = false;
+
+	if (error) {
+		return (
+			<Stack gap={1}>
+				<AlertFilledIcon color="error" size="lg" />
+				<Heading type="h2" fontSize="lg">
+					Failed to load
+				</Heading>
+				<Text>
+					There was an error loading the data. Please try refreshing the page to
+					try again.
+				</Text>
+			</Stack>
+		);
+	}
+
+	return (
+		<Stack gap={0.5}>
+			<TableWrapper>
+				<Table
+					aria-labelledby={ariaLabelledby}
+					id={tableId}
+					ref={ref}
+					tabIndex={-1}
+					tableLayout="fixed"
+				>
+					<TableHead>
+						<TableRow aria-rowindex={1}>
+							{headers.map(({ label, width }) => {
+								return (
+									<TableHeader key={label} scope="col" width={width}>
+										{label}
+									</TableHeader>
+								);
+							})}
+						</TableRow>
+					</TableHead>
+					<TableBody>
+						{loading ? (
+							<Fragment>
+								{documents.map((_, index) => (
+									<TableRow key={index}>
+										<TableCell>
+											<SkeletonText />
+											<VisuallyHidden>Loading</VisuallyHidden>
+										</TableCell>
+										<TableCell>
+											<SkeletonText />
+											<VisuallyHidden>Loading</VisuallyHidden>
+										</TableCell>
+										<TableCell>
+											<SkeletonText />
+											<VisuallyHidden>Loading</VisuallyHidden>
+										</TableCell>
+										<TableCell>
+											<SkeletonText />
+											<VisuallyHidden>Loading</VisuallyHidden>
+										</TableCell>
+										<TableCell>
+											<SkeletonBox height={32} />
+											<VisuallyHidden>Loading</VisuallyHidden>
+										</TableCell>
+									</TableRow>
+								))}
+							</Fragment>
+						) : (
+							<Fragment>
+								{documents.map((document, index) => {
+									const { id, documentType, error, file, size } = document;
+									// Adding 2 because the table header row is the first row
+									const rowIndex = index + 2;
+									return (
+										<TableRow
+											aria-rowindex={rowIndex}
+											invalid={error}
+											key={documentType}
+										>
+											<TableCell as="th" scope="row">
+												{documentType}
+											</TableCell>
+											<TableCell>
+												<Text breakWords>{file}</Text>
+											</TableCell>
+											<TableCell>
+												{error ? (
+													<StatusBadge
+														label="File missing"
+														appearance="subtle"
+														tone="errorHigh"
+													/>
+												) : (
+													size
+												)}
+											</TableCell>
+											<TableCell>
+												<Button
+													aria-label={`${
```

Comment:
Should probably use `aria-describedby` to be consistent with our other tables

Reply:
- [ ] `aria-describedby`